src/client/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -1,256 +1,74 @@
 import styled from '@emotion/styled'
 import { colorsV3, fonts } from '@hedviginsurance/brand'
 import React from 'react'
-import { QuoteDetails } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
-import {
-  apartmentTypeTextKeys,
-  getHouseholdSize,
-  insuranceTypeTextKeys,
-  maskAndFormatRawSsn,
-  quoteDetailsHasAddress,
-} from 'pages/OfferNew/utils'
-import { formatPostalNumber } from 'utils/postalNumbers'
-import { TextKeyMap, useTextKeys } from 'utils/textKeys'
+import { quoteDetailsHasAddress, isBundle } from 'pages/OfferNew/utils'
+import { useTextKeys } from 'utils/textKeys'
+import { InsuranceSummaryDetails } from './InsuranceSummaryDetails'
+import { InsuranceSummaryTermsLinks } from './InsuranceSummaryTermsLinks'
 
-const Wrapper = styled('div')`
-  padding: 0 0.5rem;
+const Wrapper = styled.div`
+  padding: 0 8px;
 `
-const Title = styled('h3')`
-  font-size: 1rem;
+const Title = styled.div`
+  font-size: 18px;
   letter-spacing: -0.23px;
   font-family: ${fonts.FAVORIT};
+  padding: 1rem 0;
 `
 
-const Table = styled('div')`
+const Table = styled.div`
   width: 100%;
+  padding-bottom: 16px;
   display: flex;
   flex-direction: column;
   font-size: 0.875rem;
   letter-spacing: -0.2px;
 `
-
-const Group = styled('div')`
-  :not(:last-of-type) {
-    margin-bottom: 1rem;
-    border-bottom: 1px solid ${colorsV3.gray300};
-  }
+export const Group = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem 0;
+  border-top: 1px solid ${colorsV3.gray300};
 `
 
-const Row = styled('div')`
+export const Row = styled.div`
+  font-size: 14px;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  padding-bottom: 1rem;
+  padding: 0.5rem 0;
   line-height: 1;
 `
-const Label = styled('div')`
-  width: 50%;
-  color: ${colorsV3.gray700};
-`
-const Value = styled('div')`
-  width: 50%;
-  color: ${colorsV3.gray700};
-  text-align: right;
-`
 
-interface Props {
+type Props = {
   offerData: OfferData
 }
 
 export const InsuranceSummary: React.FC<Props> = ({ offerData }) => {
   const textKeys = useTextKeys()
 
+  const mainQuoteInBundle = offerData.quotes.filter((quote) => {
+    return quoteDetailsHasAddress(quote.quoteDetails)
+  })
+
+  const mainQuote = isBundle(offerData)
+    ? mainQuoteInBundle[0]
+    : offerData.quotes[0]
+
```

Comment:
I saw that we have a function getting the main quote `getMainOfferQuote`, so should we use that one? It also checks for contract type  (home content). I know we looked at this together that the first quote was home contents but it feels nice to check if it's the correct type if this would change for some reason :)

Reply:
But that check is done when we get the `mainQuoteInBundle`. Or I check if quote has address, which means it's a home contents quote. But if there already is a function for this, sure I can use that.
![image](https://user-images.githubusercontent.com/42962286/114842487-2d652700-9dd9-11eb-8b9b-80b9980e3b6b.png)

Reply:
This solution is market agnostic, since it only checks for whether or not the quote has an address on it. The other solution is more specific and explicit but we need to add more logic for every market with these kinds of bundles. I'm not sure what's best, maybe the solution above makes too many assumptions?

Reply:
I hear you, this something that will grow in complexity for the more markets we add. As you say it can be that checking for address is assuming too much. We don't need to pre optimize right now, but it would be nice if we checked for the same thing in all places by using the `mainQuouteInBundle`. And we can rewrite that to check for address if you feel it makes more sense 👍

Reply:
Should we look this at a later stage in order to get it merged before tomorrow? @styrelius

Reply:
So sorry, didn't see your last comment until now. Have made the changes though 😀